src/components/networked.js

```diff
@@ -136,7 +136,7 @@ AFRAME.registerComponent('networked', {
           NAF.log.warn("Networked element was removed before ever getting the chance to syncAll");
           return;
         }
-        this.syncAll();
+        this.syncAll(undefined, true);
```

Comment:
Hmm this feels fishy. Should this just be `this.updateNextSyncTime()`? That appears to be the only thing actually run when `targetClientId` is falsey.

Reply:
Isn't it also broadcasting the initial component data to all users on connection?

Reply:
Yes, it seems that way https://github.com/MozillaReality/networked-aframe/pull/12/files#diff-69b32c2392de94b26df18f638d9c3053R208